codegenerator/cli/src/cli_args/interactive_init/evm_prompts.rs

```diff
@@ -306,16 +316,7 @@ impl ExplorerImportArgs {
         let chosen_network = match &self.blockchain {
             Some(chain) => chain.clone(),
             None => {
-                let options = NetworkWithExplorer::iter()
-                    //Filter only our supported networks
-                    .filter(|&n| {
-                        HypersyncNetwork::iter()
-                            //able to cast as u64 because networks enum
-                            //uses repr(u64) attribute
-                            .find(|&sn| n as u64 == sn as u64)
-                            .is_some()
-                    })
-                    .collect();
```

Comment:
I removed this filtering of only offering hypersync networks for contract import. It should now be obvious if they are hyper sync networks or not from the icon.